code/modules/hydroponics/hydroponics.dm

```diff
@@ -256,7 +256,8 @@
 
 //This is where stability mutations exist now.
 			if(myseed.instability >= 80)
-				mutate(0, 0, 0, 0, 0, 0, 0, 5, 0) //Exceedingly low odds of gaining a trait.
+				var/mutation_chance = 5 + (100 - myseed.instability)

```

Comment:
doesn't this lower the mutation chance the higher the instab is?

Reply:
Check the surrounding code again, we are in a special bit that is only called when instab > 80 already

Reply:
So if instab > 80
chance = 5 + (100 - instab)
plugging in some numbers,
f(x) = 5 + (100 - x)
f(81) = 5 + 19 = 24
f(99) = 5 + 1 = 6
This doesn't sound like the behavior you want

Reply:
wooooooooppppppppppppsssssssssssssssssssssss